pkg/util/test/util.go

```diff
@@ -3232,6 +3216,51 @@ func validateAllStorageNodesInState(namespace string, status corev1.NodeConditio
 	}
 }
 
+// GetImageURN returns the complete image name based on the registry and repo
+func GetImageURN(cluster *corev1.StorageCluster, image string) string {
```

Comment:
Let's not copy this function, in the future if we change this function, test would not catch it.  you can write a function ValidateImage, for 1.9.1 call util.ImageURN, for lower version if image does not match, remove docker.io prefix and compare again.

Reply:
I wanted to add the check for operator version inside `GetImageURN()` in `pkg/util/util.go` but couldn't due to import issue